build-common.properties

```diff
@@ -46,6 +46,7 @@ dataflow.lib=${dataflow.loc}/dist/dataflow.jar
 dataflow.sources.lib=${dataflow.loc}/dist/dataflow-source.jar
 dataflow.javadoc.lib=${dataflow.loc}/dist/dataflow-javadoc.jar
 
+afu.lib=${annotation.tools}/annotation-file-utilities/annotation-file-utilities.jar
```

Comment:
"afu.lib" should be with "annotation.tools" in the external dependency section.
For consistency, I would use a longer name.